src/action/common/configure_init_service.rs

```diff
@@ -157,23 +161,38 @@ impl Action for ConfigureInitService {
                 )
                 .await?;
 
-                execute_command(
-                    Command::new("systemctl")
-                        .process_group(0)
-                        .arg("link")
-                        .arg(SERVICE_SRC)
-                        .stdin(std::process::Stdio::null()),
-                )
-                .await?;
+                if Path::new(SERVICE_DEST).exists() {
```

Comment:
We could also add this check to the `plan` and provide an error that has remediation instructions, e.g. "remove the existing blah blah blah with blah blah blah", but that seems kinda like not-curing? I dunno, happy to take whichever path seems best.

Reply:
It's perfectly okay to bail on things we can't cure right now! My rule has been to give a command for them to run in the error. We may rework the UX a bit later to offer some prompting of the user, but that doesn't exist yet.